sql/telemetry/clients_aggregates_v1.sql

```diff
@@ -0,0 +1,166 @@
+WITH scalars_and_booleans AS
+  -- Aggregate per client without date.
+  (SELECT
+    client_id,
+    os,
+    app_version,
+    app_build_id,
+    channel,
+    aggregate.metric as metric,
+    aggregate.metric_type AS metric_type,
+    aggregate.key AS key,
+    aggregate.agg_type as agg_type,
+    CASE agg_type
+      WHEN 'max' THEN max(value) OVER w1
+      WHEN 'min' THEN min(value) OVER w1
+      WHEN 'avg' THEN avg(value) OVER w1
+      WHEN 'sum' THEN sum(value) OVER w1
+      WHEN 'false' THEN sum(value) OVER w1
+      WHEN 'true' THEN sum(value) OVER w1
+    END AS agg_value
+  FROM
+    clients_daily_scalar_aggregates_v1
+  CROSS JOIN
+    UNNEST(scalar_aggregates) AS aggregate
+  WHERE value IS NOT NULL
+  WINDOW
+      -- Aggregations require a framed window
+      w1 AS (
+          PARTITION BY
+              client_id,
+              os,
+              app_version,
+              app_build_id,
+              channel,
+              aggregate.metric,
+              aggregate.metric_type,
+              aggregate.key,
+              aggregate.agg_type)
+  ),
+
+keyed_scalars AS
+  (SELECT
+      client_id,
+      os,
+      app_version,
+      app_build_id,
+      channel,
+      aggregate.metric as metric,
+      aggregate.metric_type AS metric_type,
+      aggregate.key AS key,
+      aggregate.agg_type as agg_type,
+      CASE agg_type
+        WHEN 'max' THEN max(value) OVER w1
+        WHEN 'min' THEN min(value) OVER w1
+        WHEN 'avg' THEN avg(value) OVER w1
+        WHEN 'sum' THEN sum(value) OVER w1
+        WHEN 'false' THEN sum(value) OVER w1
+        WHEN 'true' THEN sum(value) OVER w1
+      END AS agg_value
+    FROM
+      clients_daily_keyed_scalar_aggregates_v1
+    CROSS JOIN
+      UNNEST(scalar_aggregates) AS aggregate
+    WHERE value IS NOT NULL
+    WINDOW
+        -- Aggregations require a framed window
+        w1 AS (
+            PARTITION BY
+                client_id,
+                os,
+                app_version,
+                app_build_id,
+                channel,
+                aggregate.metric,
+                aggregate.metric_type,
+                aggregate.key,
+                aggregate.agg_type)
+  ),
+
+all_booleans AS
+    (SELECT *
+    FROM scalars_and_booleans
+    WHERE metric_type = "boolean"
+
+    UNION ALL
+
+    SELECT *
+    FROM keyed_scalars
+    WHERE metric_type = "keyed-scalar-boolean"),
```

Comment:
OH! You actually made me realize a mistake... 
There should only be one table called `clients_daily_scalar_aggregates_v1` which contains keyed scalars as well. So this would naturally be less redundant.

Thank you!!